doc/object_tracing.rst

```diff
@@ -0,0 +1,23 @@
+.. highlight:: rest
+
+**************
+Object Tracing
+**************
+
+This document describes how the code traces
+each object found within a slit/order.
```

Comment:
let's change `slit/order` to `slit` throughout this doc., since `order` is used for a polynomial order later. Also, I've tried to call most things "slits" throughout the echelle pipeline for consistency.